docs/docs/guides/docker.md

```diff
@@ -0,0 +1,61 @@
+---
+id: docker
+title: Docker Images
+---
+## Supported tags and respective `Dockerfile` links
+* [`latest`, `v0.4.3-alpha.1`, `v0.4.3`, `v0.4`, `v0`](https://github.com/ory/kratos/blob/v0.4.3-alpha.1/.docker/Dockerfile)
+* [`latest-sqlite`, `v0.4.3-alpha.1-sqlite`, `v0.4.3-sqlite`, `v0.4-sqlite`, `v0-sqlite`](https://github.com/ory/kratos/blob/v0.4.3-alpha.1/.docker/Dockerfile-sqlite)
+
+## Image Variants
+The `Kratos` Docker images come in two different falvors, one with and one without SQLite support. All Docker images with the postfix `kratos:<version>-sqlite` in the tag are compiled with embed SQLite support and uses libmusl. All Docker images (`kratos:<version>`) without the postfix `-sqlite` are compiled without SQLite support and therefore also don't include libmusl. 
+
+If you don't make use of the embedded SQLite support we recommend to use the Docker images without SQLite support as they are smaller in size, include fewer libraries and therefore have a smaller attack surface. 
+
+## How to use these images
+In order to make the provided Docker images as useful as possible they can be configured through a set of supported Environment variables. In addition the default configuration directoy can be bound to a directory of choice to make it simple to pass in your own configuration files.
+
+### Environment Variables
+#### `DSN`
+This environment variable allows you to specify the database source name. As the `DSN` normally consists of the url to the database system and the credentials to access the database it is recommended to specify the `DSN` using a Environment variable. 
+
+**Example:** 
+
+`docker run -e DSN="memory" oryd/kratos:latest`
+
+
+#### `SECRETS_DEFAULT`
+This environment variable allows you to specify the .....
+
+**Example:** 
+
+`docker run -e SECRETS_DEFAULT="supersecretkey" oryd/kratos:latest`
```

Comment:
If you add this to the [tag update](https://github.com/ory/kratos/blob/master/docs/config.js#L8) automation and I also recommend using a secret that has `change me` as a content to make it obvious that one should change it :)

```suggestion
`docker run -e SECRETS_DEFAULT="CHANGE-ME" oryd/kratos:v0.4.3-alpha.1`
```